### DISCUSSION TITLE
export solution to Git  - pac - Clone or Sync Solution Error

### DISCUSSION THREAD

**Discussion by mrramosjr21** [microsoft/powerplatform-build-tools#752] [General]
Hi All,
Good Day!

Does anyone encountered this error when exporting solution to devOps **create a branch from main to new branch.**

We are using a customize ALM in our company and this is the error I'm receiving everytime we do a export solution to git.
We are checking the path and the path doesn't exist anywhere so I'm hoping someone can help me about this.

Below is the exact error.

Unpacking Solution...

Extracting C:\Users\VssAdministrator\AppData\Local\Temp\OutboundSupplierRegistrationApp_1387.zip and C:\Users\VssAdministrator\AppData\Local\Temp\OutboundSupplierRegistrationApp_1387_managed.zip to D:\a\1\s\OutboundSupplierRegistrationApp_1387\OutboundSupplierRegistrationApp_1387\SolutionPackage\OutboundSupplierRegistrationApp_1387\src

Skipping localization
Processing Component: Workflows
 - 1387_1Refresh Minutos
 - 1387_Guardar informacion en historico
 - 1387_1Alerta de rechazo
 - 1387_1Enviar alerta en Teams
Processing Component: CanvasApps
 - otd1387_vdctkm17accesscontrol_a394c
Microsoft PowerPlatform CLI
Version: 1.30.7+ge6211cf
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

**Error: Could not find a part of the path 'D:\a\1\s\OutboundSupplierRegistrationApp_1387\OutboundSupplierRegistrationApp_1387\SolutionPackage\OutboundSupplierRegistrationApp_1387\src\environmentvariabledefinitions\otd1387_shared_sharepointonline_7db39709998143d4a4dccdcf8eb305bb\environmentvariabledefinition.xml'.**

Moving items from D:\a\1\s\OutboundSupplierRegistrationApp_1387\OutboundSupplierRegistrationApp_1387\SolutionPackage\OutboundSupplierRegistrationApp_1387 to D:\a\1\s\OutboundSupplierRegistrationApp_1387\OutboundSupplierRegistrationApp_1387\SolutionPackage
Removing redundant folder D:\a\1\s\OutboundSupplierRegistrationApp_1387\OutboundSupplierRegistrationApp_1387\SolutionPackage\OutboundSupplierRegistrationApp_1387
##[error]PowerShell exited with code '1'.
Finishing: pac - Clone or Sync Solution

Thank you in advance

**Comment by shefali-bedi** [comment]
Hi, i am facing same problem with different error. If i run my script from the that new branch , it is working. Or if i am running command from VS code it works while creating a new branch but not on subsequents runs. It is extremely confusing. Thought it is good to connect in case any one of us finds a solution

**Comment by mikefactorial** [comment]
The error message is confusing but what you're seeing is a windows long path error since the path 
```
D:\a\1\s\OutboundSupplierRegistrationApp_1387\OutboundSupplierRegistrationApp_1387\SolutionPackage\OutboundSupplierRegistrationApp_1387\src\environmentvariabledefinitions\otd1387_shared_sharepointonline_7db39709998143d4a4dccdcf8eb305bb\environmentvariabledefinition.xml
```
is greater than 260 characters. You can use long path notation on sync / clone (e.g. \\?\D:\very long path) to get around this issue as mentioned here https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry
